package.json

```diff
@@ -52,7 +52,9 @@
 	},
 	"dependencies": {},
 	"devDependencies": {
+		"cldr-data": ">=26",
```

Comment:
I assume we would need at least the locales defined here: https://github.com/jquery/jquery-ui/pull/1651/files#diff-35b4a816e0441e6a375cd925af50752cR408